src/mlpack/methods/reinforcement_learning/environment/mountain_car.hpp

```diff
@@ -177,6 +177,20 @@ class MountainCar
     return Sample(state, action, nextState);
   }
 
+  /**
+   * Dynamics of Mountain Car. Get reward based on current state and current
+   * action.
+   *
+   * @param state The current state.
+   * @param action The current action.
+   * @return reward, it's always -1.0.
+   */
+  double Sample(const State& state, const Action& action, State& nextState,
+                 const State& transitionGoal)
+  {
```

Comment:
Comment out `transitionGoal`, I'm sure you've seen others do it 

```
double Sample(const State& state, const Action& action, State& nextState,
              const State& /* transitionGoal */)
```

Reply:
Can you explain this a bit ?